ui/pages/setup_secure.tsx

```diff
@@ -20,11 +20,7 @@ const SetupSecure: NextPage = () => {
 
   // Get ?flow=... from the URL
   const router = useRouter();
-  const {
-    return_to: returnTo,
-    flow: flowId,
-    pw_changed: pwChanged,
-  } = router.query;
+  const { return_to, flow: flowId, pw_changed: pwChanged } = router.query;
```

Comment:
```suggestion
  const { return_to: returnTo, flow: flowId, pw_changed: pwChanged } = router.query;
```
Sorry, my previous comment was ambiguous, what I meant is to keep the old alias `returnTo` in favour of changing it to `_returnTo`. We like the camlCase and dislike snake_case in tsx code.